clients/v1_repl/v1_repl.go

```diff
@@ -0,0 +1,723 @@
+		db = c.Database
+		rp = c.RetentionPolicy
+		point = getIdentFromMatches(insertMatches, "point")
+		if db == "" {
+			color.Red("Please run \"use <database>\" to run \"INSERT <point\"")
+			return
+		}
+	} else {
+		color.Red("Expected \"INSERT INTO <database>.<retention_policy> <point>\" OR \"INSERT <point>\".")
+		return
+	}
+
+	buf := bytes.Buffer{}
+	gzw := gzip.NewWriter(&buf)
+
+	_, err := gzw.Write([]byte(point))
+	gzw.Close()
+	if err != nil {
+		color.Red("Failed to gzip points")
+		return
+	}
+	bucketID, err := c.getBucketIdForDBRP(db, rp)
+	if err != nil {
+		color.Red("Unable to match DBRP to BucketID: %v", err)
+		return
+	}
+	writeReq := c.PostWrite(context.Background()).
+		Bucket(bucketID).
+		Precision(api.WritePrecision(c.Precision)). // TODO: fix, rfc3339 wouldn't be a valid writePrecision but is a valid query precision
+		ContentEncoding("gzip").
+		Body(buf.Bytes())
+	if c.OrgID != "" {
+		writeReq = writeReq.Org(c.OrgID)
+	} else if c.OrgName != "" {
+		writeReq = writeReq.Org(c.OrgName)
+	} else {
+		writeReq = writeReq.Org(c.ActiveConfig.Org)
+	}
+	if err := writeReq.Execute(); err != nil {
+		color.Red("ERR: %v", err)
+		if c.Database == "" {
+			fmt.Println("Note: error may be due to not setting a database or retention policy.")
+			fmt.Println(`Please set a database with the command "use <database>"`)
+			return
+		}
+	}
+}
+
+// Reverse search for a bucket from db & rp
+func (c Client) getBucketIdForDBRP(db string, rp string) (string, error) {
+	dbrpsReq := c.GetDBRPs(context.Background())
+	if c.OrgID != "" {
+		dbrpsReq = dbrpsReq.OrgID(c.OrgID)
+	}
+	if c.OrgName != "" {
+		dbrpsReq = dbrpsReq.Org(c.OrgName)
+	}
+	if c.OrgID == "" && c.OrgName == "" {
+		dbrpsReq = dbrpsReq.Org(c.ActiveConfig.Org)
+	}
+	dbrps, err := dbrpsReq.Execute()
+	if err != nil {
+		return "", err
+	}
+	bucketID := ""
+	for _, dbrp := range *dbrps.Content {
+		if dbrp.Database == db && dbrp.RetentionPolicy == rp {
+			bucketID = dbrp.BucketID
+			break
+		}
+	}
+	if bucketID == "" {
+		return "", fmt.Errorf("unable to find bucket ID for %q.%q", db, rp)
+	}
+	return bucketID, nil
+}
+
+type FormatType string
+
+var (
+	CsvFormat   FormatType = "csv"
+	JsonFormat  FormatType = "json"
+	TableFormat FormatType = "table"
+)
+
+func (c *Client) runAndShowQuery(query string) {
+	// TODO: guide users trying to use deprecated InfluxQL queries
+	response, err := c.query(context.Background(), query)
+	if err != nil {
+		color.HiRed("Query failed.")
+		color.Red("%v", err)
+		return
+	}
+	displayMap := map[FormatType]func(string){
+		CsvFormat:   c.outputCsv,
+		JsonFormat:  c.outputJson,
+		TableFormat: c.outputTable,
+	}
+	displayFunc := displayMap[c.Format]
+	displayFunc(response)
+}
+
+func (c *Client) help() {
+	fmt.Println(`Usage:
+		pretty                toggles pretty print for the json format
+        use <db_name>         sets current database
+        format <format>       specifies the format of the server responses: json, csv, or column
```

Comment:
Should be `table` not `column`

Reply:
Oh, actually, I think column is the 1.x version of table. I'll implement that. That could also be a good fallback format for non-tty, what do you think?